runtime/api/src/main/java/org/qbicc/runtime/CNative.java

```diff
@@ -134,7 +134,13 @@ private CNative() {
      * @param <T> the object type
      * @return a pointer to the object
      */
-    public static native <T extends object, P extends ptr<T>> P addr_of(T obj);
+    public static native <T, P extends ptr<T>> P addr_of(T obj);
+
+    /**
+     * @hidden
+     */
+    @Deprecated
+    public static native <T extends object, P extends ptr<T>> P addr_of$$bridge(T obj);
```

Comment:
This is the bridge method.  It allows code compiled against the version of `addr_of` which accepts `object` instead of `Object` to continue to run, but new code will compile against the `Object` variant because synthetic bridge methods are hidden from `javac`.